src/einundzwanzig.py

```diff
@@ -45,4 +48,39 @@ def episode(update: Update, context: CallbackContext):
     else:
         message = 'Das ist kein gültiges Podcast-Format! Bitte gibt eins der folgenden Formate an: Alle, Interviews, Lesestunde, News, Weg'
 
-    context.bot.send_message(chat_id=update.effective_chat.id, text=message)
\ No newline at end of file
+    context.bot.send_message(chat_id=update.effective_chat.id, text=message)
+
+def getInvoice(amt, shoutout):
+    config.TALLYDATA['satoshi_amount'] = str(amt)
+    config.TALLYDATA['message'] = shoutout[0:139]
+    response = requests.post('https://api.tallyco.in/v1/payment/request/', data=config.TALLYDATA).text
+    dict = json.loads(response)
+    return dict.get("lightning_pay_request")
+
+def createQR(text):
+    img = qrcode.make(text)
+    img.save('qr.png')
```

Comment:
Is it necessary to save that image or can we keep that object in memory and send it from there? We also need to be careful about concurrency issues here.